components/action-mgt/org.wso2.carbon.identity.action.management/pom.xml

```diff
@@ -154,7 +184,7 @@
                 <groupId>com.github.spotbugs</groupId>
                 <artifactId>spotbugs-maven-plugin</artifactId>
                 <configuration>
-                    <threshold>High</threshold>
+                    <skip>true</skip>
```

Comment:
why this is changed ?
Keep spot bugs original threshold

Reply:
Updated in